years_to_hours.rb

```diff
@@ -0,0 +1,26 @@
+def ask(question)
+	print question + " "
+	gets.chomp 
+end
```

Comment:
I like this idea of defining it's own method for the questions so you can re-use it. Nice!